config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
   constraints defaults: { format: :json } do
     api version: 1, default: true do
+      resources :users, only: [:create, :destroy] do
+        collection do
+          patch   :update
```

Comment:
ale z założenia user nie może edytować innego konta, więc jeżeli dodam `:id` to muszę jeszcze sprawdzić czy `@user == @current_user`. User może edytować jedno konto przez co nie powinien podawać `:id` analogicznie jest przy logout: podczas usuwania sesji nie podajesz jej `:id`